docs/java-rest/sniffer.asciidoc

```diff
@@ -8,7 +8,14 @@ Nodes Info api and uses jackson to parse the obtained json response.
 
 Compatible with Elasticsearch 2.x and onwards.
 
-=== Maven Repository
+The low-level REST client is subject to the same release cycle as
```

Comment:
This change (Removing the title) moves this section to the previous page I think. Is this what you wanted to do? Should we have the same type of maven central introduction that we have for the rest client in the other docs page?